nx.json

```diff
@@ -66,7 +66,7 @@
 			"!{projectRoot}/tsconfig.storybook.json",
 			"!{projectRoot}/eslint.config.js"
 		],
-		"sharedGlobals": []
+		"sharedGlobals": ["{workspaceRoot}/package.json", "{workspaceRoot}/bun.lockb"]
```

Comment:
What does this do?

Reply:
If those files change, nx cache is invalidated